### PR TITLE
XFA - Remove the checked attribute from the checkbox when unchecked (bug 1729877)

### PR DESCRIPTION
**Pull request by calixteman**
- it aims to fix: https://bugzilla.mozilla.org/show_bug.cgi?id=1729877.

### CODE REVIEW

File: test/test_manifest.json [hunk 1]
```diff
@@ -967,6 +967,21 @@
        "enableXfa": true,
        "type": "eq"
     },
+    {  "id": "xfa_bug1729877",
+       "file": "pdfs/xfa_bug1729877.pdf",
+       "md5": "1bf5429ea13a090f52edcd49cb0c4a47",
+       "link": true,
```
Comment (Snuffleupagus) [inline test/test_manifest.json:973]: Are we allowed to include this document in the repo?
 - If no, please update the patch to actually use a *linked* test-case instead.
 - If yes, this line should simply be removed.

**Review by marco-c — APPROVED** [review]
r+, with @Snuffleupagus's comment addressed

**Pull request merged**